src/main/java/org/junitpioneer/jupiter/ApplyMode.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2016-2022 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
```

Comment:
Javadoc :)

Reply:
Javadoc and normal documentation is pending, I just wanted to check if this approach is OK, first.